src/dedupe.coffee

```diff
@@ -71,6 +72,7 @@ class Dedupe extends Command
 
   forkDedupeCommand: (options, callback) ->
     dedupeArgs = ['--globalconfig', config.getGlobalConfigPath(), '--userconfig', config.getUserConfigPath(), 'dedupe']
+    dedupeArgs.push("--runtime=electron")
```

Comment:
I believe the variable is called `dedupeArgs` because it is the list of arguments passed to `npm dedupe`